.gitignore

```diff
@@ -4,8 +4,6 @@
 
 packages/
 
-inst/lib/

```

Comment:
@PavelBal I am adding back those dlls. This will simplify getting started, at least on windows. No need to install anything. 
The build script will always update the dlls but this should be equivalent